code/outils.sls

```diff
@@ -0,0 +1,142 @@
+
+(library (outils)
+  (export lookup
+          lookup-partial
```

Comment:
Do you want to go ahead and consolidate these two functions with a case-lambda, or would you prefer to leave that for another PR?